docs/PLUGIN_HOOKS.md

```
@@ -1118,6 +1118,88 @@ Permette di aggiungere voci al menu frontend.
 
 ---
 
+## Hook di Integrazione (usati dai plugin bundled)
+
+Questi hook sono **effettivamente invocati dal core** e usati dai plugin distribuiti con Pinakes (archives, oai-pmh-server, frbr-lrm, ecc.).
+
+### `app.routes.register` (Action)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)